analysis/R/association.R

```diff
@@ -377,13 +380,31 @@ ComputeDistributionEM <- function(reports, report_cohorts, maps,
     Log('\tGetCondProb for each report (%d cores)', num_cores)
 
     # Get the joint conditional distribution
-    cond_report_dist <- mclapply(seq(length(var_report)), function(i) {
-      cohort <- var_cohort[i]
-      #Log('Report %d, cohort %d', i, cohort)
-      bit_indices <- bit_indices_by_cohort[[cohort]]
-      GetCondProb(var_report[[i]], pstar, qstar, bit_indices,
-                  prob_other = prob_other[[cohort]])
-    }, mc.cores = num_cores)
+    # TODO(pseudorandom): check RAPPOR type more systematically instead of by
+    # checking if k == 1
+    if(k == 1) {
```

Comment:
Can you separate this out into two functions, for boolean and string?  Because we will need another one for Basic RAPPOR, right?